datalad/support/tests/test_sshconnector.py

```diff
@@ -167,3 +170,22 @@ def test_ssh_copy(sourcedir, sourcefile1, sourcefile2):
             eq_(content, fp.read())
 
     ssh.close()
+
+
+@skip_ssh
+def test_ssh_compound_cmds():
+    ssh = SSHManager().get_connection('ssh://localhost')
+    out, err = ssh('[ 1 = 2 ] && echo no || echo success')
```

Comment:
great -- I just was hinting on it would be great if it was tested at least a bit here on that regard